src/main/main/java/Duke.java

```diff
@@ -32,7 +32,7 @@ public static void main(String[] args) {
                 System.out.println("____________________________________________________________\n"
                         + "Here are the tasks in your list:");
                 for(int i = 0; i < listCount; i++){
```

Comment:
Coding, standard. I think there should be space between "for" and braket, as well as the bracket and curly bracket